de.dlr.sc.virsat.model.edit/src/de/dlr/sc/virsat/model/concept/types/structural/IBeanStructuralElementInstance.java

```diff
@@ -104,6 +105,18 @@
 	 */
 	<SEI_TYPE extends IBeanStructuralElementInstance> List<SEI_TYPE> getChildren(Class<SEI_TYPE> beanSeiClazz);
 	
+	/**
+	 * This method hands back all child Bean wrapped StructuralElementInstances of the abstract type ABeanStructuralElementInstance
+	 * @return all children which could be found in the current Bean SEI
+	 */
+	List<ABeanStructuralElementInstance> getChildren();

```

Comment:
Using the abstract bean class in the interface declaration seems weird to me... Maybe we can consistently use the IBeanStructuralElementInstance everywhere?

Reply:
As we talked about, we can't easily